agent/main.go

```diff
@@ -226,6 +229,28 @@ func main() {
 		return
 	}
 
+	// if secure-access is enabled create csr
+	if feature.Gates.Enabled(feature.SecureAccess) {
+		logger.Info("creating host csr")
+		byohCSR := registration.ByohCSR{K8sClient: k8sClient}
+		// TODO: persist this key across host-agent restarts
```

Comment:
Hopefully there is an issue to track it beyond the PR

Reply:
Created #519 to track this